### ISSUE TITLE
arm64 v8 pause container not matched

### ISSUE THREAD

**Issue by sdenovan** [containers/podman#6507]
/kind bug

**Description**

**Steps to reproduce the issue:**

1. Install Ubuntu 20.04 (arm64) on a Raspberry Pi 4

2. Install Podman using the Ubuntu instructions (apt integration)

3. podman pod create

**Describe the results you received:**
```
# podman pod create                   
ERRO[0000] Error freeing pod lock after failed creation: no such file or directory 
Error: unable to create pod: error adding Infra Container: unable to pull k8s.gcr.io/pause:3.2: unable to pull image: Error choosing an image from manifest list docker://k8s.gcr.io/pause:3.2: no image found in manifest list for architecture arm64, variant v8, OS linux
```

**Describe the results you expected:**
Pod created

**Additional information you deem important (e.g. issue happens only occasionally):**
Looking at the skopeo output of:
```
skopeo inspect --raw docker://k8s.gcr.io/pause:3.2 | jq .
```
I noticed that the arm64 pause container does not specify a variant:
```
    {
      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
      "size": 526,
      "digest": "sha256:31d3efd12022ffeffb3146bc10ae8beb890c80ed2f07363515580add7ed47636",
      "platform": {
        "architecture": "arm64",
        "os": "linux"
      }
    },
```
My guess is that the "v8" variant is preventing a match for the pause container but I'm not having any luck finding a way to override it. An `--override-variant` option might work here, but perhaps the absence of a variant specification on the upstream container should match all variants? That seems to be the approach being advocated here:

[https://github.com/containers/image/issues/898](url)
 
**Output of `podman version`:**

```
Version:            1.9.3
RemoteAPI Version:  1
Go Version:         go1.13.8
OS/Arch:            linux/arm64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  gitCommit: ""
  goVersion: go1.13.8
  podmanVersion: 1.9.3
host:
  arch: arm64
  buildahVersion: 1.14.9
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.16, commit: '
  cpus: 4
  distribution:
    distribution: ubuntu
    version: "20.04"
  eventLogger: file
  hostname: REDACTED
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 5.4.0-1011-raspi
  memFree: 30969856
  memTotal: 3977601024
  ociRuntime:
    name: runc
    package: 'runc: /usr/sbin/runc'
    path: /usr/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  os: linux
  rootless: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 0
  swapTotal: 0
  uptime: 220h 11m 50.06s (Approximately 9.17 days)
registries:
  search:
  - docker.io
  - quay.io
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /var/lib/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 2
  runRoot: /var/run/containers/storage
  volumePath: /var/lib/containers/storage/volumes

```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/unknown 1.9.3~1 amd64
podman/unknown,now 1.9.3~1 arm64 [installed]
podman/unknown 1.9.3~1 armhf
podman/unknown 1.9.3~1 s390x
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by mtrmac** [comment]
Thanks for your report; this should be fixed in Podman 2.0.0, which has been released since. Can you retest, please?

**Comment by rhatdan** [comment]
Closing, reopen if it is not fixed.
